app/tests/v2/sample_data.py

```diff
@@ -86,10 +86,10 @@
 
 NEW_MEETUP = dict(
     location="Test New Location",
-    images=[],
+    images='{}',
     topic="Test New Topic",
     happeningOn="Mar 5 2019 11:00AM",
-    tags=["Programming", "Design"]
+    tags='{Programming, Design}'
```

Comment:
Just thinking, tags should be a list ([...]) and not Objects or Dictionaries ({...}). Any reason why opted to use the object?

Reply:
Please refer to this PR #22. I have used the Postgresql syntax for array of strings. When I used the Python list of syntax, I was encountering a malformed syntax error.